cockatrice/src/tab_deck_editor.cpp

```diff
@@ -764,6 +771,19 @@ void TabDeckEditor::actPrintDeck()
     dlg->exec();
 }
 
+void TabDeckEditor::actExportDeckDecklist()
+{
+    DeckLoader *const deck = deckModel->getDeckList();
+    if(deck != NULL){
+        QString decklistUrlString = deck->exportDeckToDecklist();
+        QUrl decklistUrl = QUrl(decklistUrlString);
+        decklistUrlString = decklistUrl.toEncoded();
```

Comment:
Hmm, does this come out right? You're adding explicit `%20` into the string, wouldn't that double-encode the `%`?

If you're feeling adventurous, a unit test for this case would be great, but we'll accept the PR without it - the infrastructure for writing them is fairly confusing here.

Reply:
Actually it does come out right - the QUrl parser is smart enough to know that it doesn't need to encode it if it's a % followed by two numbers already.

I'll try my hand at a unit test, but that may be a while (as it doesn't sound exceptionally interesting, considering i'd mainly be testing the QUrl .toEncoded function which I'm 99.99% sure is just fine). Accepting this pr before I get that done may be a good idea.